### ISSUE TITLE
ANN chapter

### ISSUE THREAD

**Issue by bpesquet** [bpesquet/mlhandbook#11]
- [x] Hide useless cells
- [x] Universal approx theorem
- [ ] SELU and leaky RELU
- [ ] GD optim viz ([source](https://imgur.com/a/Hqolp#NKsFHJb))
- [ ] Backprop demo
- [ ] Batch norm
- [ ] Geometric intuitions (DMxUCL 2020 l2, Ng, Olah)
